docs/src/debugger.md

```diff
@@ -1,4 +1,4 @@
-# Debugger (Beta)
```

Comment:
We prolly need to take it off of the content list (plus, it's inconsistent: the content list says "Debugging" and here it's "Debugger")